packages/tizen_app_control/example/integration_test/tizen_app_control_test.dart

```diff
@@ -26,6 +26,16 @@ void main() {
     expect(received.operation, 'http://tizen.org/appcontrol/operation/default');
   }, timeout: kTimeout);
 
+  testWidgets('Can find matching applications', (WidgetTester _) async {
+    final AppControl request = AppControl(
+      operation: 'http://tizen.org/appcontrol/operation/view',
+      uri: 'file:///opt/usr/globalapps/$kAppId/shared/res/ic_launcher.png',
+      mime: 'image/*',
+    );
+    final List<String> matches = await request.getMatchedAppIds();
+    expect(matches, isNotEmpty);
+  }, timeout: kTimeout);
```

Comment:
This test should not be run on TV emulators. I couldn't find any operation that is commonly available on both wearable and TV emulators.

Reply:
how about tagging this test to mark "wearable only"?
https://pub.dev/packages/test#tagging-tests

Reply:
Could you elaborate how to configure? How can the test runner detect the current device profile at runtime?